examples/deleonberne_hamiltonian.py

```diff
@@ -0,0 +1,313 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 30 10:02:48 2019
+@author: Wenyang Lyu and Shibabrat Naik
+Script to define expressions for the coupled quartic Hamiltonian
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.integrate import solve_ivp
+import time
+import math
+from scipy import optimize
+
+
+#% Begin problem specific functions
+def init_guess_eqpt_deleonberne(eqNum, par):
+    """
+    Returns configuration space coordinates of the equilibrium points according to the index:
+    Saddle (EQNUM=1)
+    Centre (EQNUM=2,3)
+    """
+    
+    if eqNum == 1:
+        x0 = [0, 0]
```

Comment:
Having an `if` like this feels wrong. Why don't you just return e.g. a list of the three points?